kolibri/plugins/management/assets/src/state/addCoachRoleAction.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable prefer-arrow-callback */
+import * as coreApp from 'kolibri';
+import * as constants from 'kolibri.coreVue.vuex.constants';
```

Comment:
By convention we've been avoiding `import` and `export` statements in our code base, and sticking to `require` and `module.exports`.

This was done because the webpack 1.0 ES6 `import` syntax didn't have support for asynchronous code splitting. I think we should probably stick with `require` for consistency, though I'm a fan of moving to `import`/`export` eventually...